.circleci/config.yml

```diff
@@ -668,12 +668,9 @@ jobs:
       - run:
           name: Build development contracts
           command: |
-            for contract_dir in ./contracts/*/; do
-              name=$(basename $contract_dir)
-              echo "Building $name ..."
-              docker run --volumes-from with_code cosmwasm/rust-optimizer:0.10.5 ./contracts/$name
-              docker cp with_code:/code/artifacts/$name.wasm ./artifacts/$name.wasm
-            done
+            echo "Building all contracts under ./contracts"
+            docker run --volumes-from with_code cosmwasm/rust-optimizer:0.10.6 ./contracts/*/
+            docker cp with_code:/code/artifacts/*.wasm ./artifacts
```

Comment:
Could we copy `checksums.txt` from the guest as well? Then we can remove the next step ("Create checksums").